### PR TITLE
Fix exchange balance amount bug

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.
- [ ] Fix exchange balance amount bug (https://discord.com/channels/657906918408585217/683245846070165504/965725468173168701)

### CODE REVIEW

File: frontend/app/src/components/AssetBalances.vue [hunk 1]
```diff
@@ -26,9 +26,8 @@
     </template>
     <template #item.usdValue="{ item }">
       <amount-display
-        show-currency="symbol"
-        :fiat-currency="item.asset"
-        :amount="item.amount"
+        :fiat-currency="'USD'"
```
Comment (LefterisJP) [inline frontend/app/src/components/AssetBalances.vue:29]: why are you hard coding USD here?

**Review by LefterisJP — COMMENTED** [review]
I tested it. In the bittrex exchange view it's fine now, but the same problem exists in the dashboard

File: frontend/app/src/components/AssetBalances.vue (hunk 1 above)
Comment (lukicenturi) [reply frontend/app/src/components/AssetBalances.vue]: It is needed for the amount-display convert the value to selected currency (USD indicate what currency used for the amount passed, in this case it is always USD value)

I think this part was not checked, when we update the <amount-display /> component

Comment (lukicenturi) [reply frontend/app/src/components/AssetBalances.vue]: Dashboard updated too

Comment (lukicenturi) [reply frontend/app/src/components/AssetBalances.vue]: wait there are other part i need to fix too

Comment (lukicenturi) [reply frontend/app/src/components/AssetBalances.vue]: I've fix several wrong implementation. It should be correct now

**Review by LefterisJP — APPROVED** [review]
LGTM

**Comment by lukicenturi** [comment]
The e2e test failed because there are bug when inputing Fiat Asset (i.e EUR). I'm fixing it... I think I need to restructure the <amount-display /> since it is kinda messy now.

**Comment by LefterisJP** [comment]
> The e2e test failed because there are bug when inputing Fiat Asset (i.e EUR). I'm fixing it... I think I need to restructure the since it is kinda messy now.

But does it pass locally? Or merging this will break e2e for develop?

**Comment by lukicenturi** [comment]
> > > The e2e test failed because there are bug when inputing Fiat Asset (i.e EUR). I'm fixing it... I think I need to restructure the since it is kinda messy now.
> > 
> > 
> > But does it pass locally? Or merging this will break e2e for develop?
> 
> No it's not passed locally. The test is doing the right job. I'll merge it later when this bug is fixed.

The test failed here is not the flaky one, it's different test

**Comment by LefterisJP** [comment]
> > > > The e2e test failed because there are bug when inputing Fiat Asset (i.e EUR). I'm fixing it... I think I need to restructure the since it is kinda messy now.
> > > 
> > > 
> > > But does it pass locally? Or merging this will break e2e for develop?
> > 
> > 
> > No it's not passed locally. The test is doing the right job. I'll merge it later when this bug is fixed.
> 
> The test failed here is not the flaky one, it's different test

Please be checking discord as for such conversation it's better to discuss there.

**Comment by lukicenturi** [comment]
@LefterisJP I'm gonna merge this PR, bug I mentioned earlier is fixed

**Pull request merged**